mesa/visualization/solara_viz.py

```diff
@@ -235,12 +235,12 @@ def save_initial_model():
 
     solara.use_effect(save_initial_model, [model.value])
 
-    def step():
+    async def step():
         while playing.value:
-            time.sleep(play_interval / 1000)
+            await asyncio.sleep(1 / 1000)
```

Comment:
Shouldn't `1` be `play_interval`?

Reply:
oops, of course. Thats why we have code reviews I guess :)